environment-api/src/main/java/com/sequenceiq/environment/api/doc/environment/EnvironmentModelDescription.java

```diff
@@ -103,6 +103,7 @@
             + "or use multiple resource groups.";
 
     public static final String ENCRYPTION_KEY_URL = "URL of the CustomerManagedkey to encrypt Azure resources";
+    public static final String DISK_ENCRYPTION_SET_ID = "Resource Id of the encryption set used to encrypt Azure resources.";
```

Comment:
Please be a bit more specific: "Azure disks" or "Azure disk resources". Also, insert "disk" before "encryption".